src/reader/ArticleReader.js

```diff
@@ -108,10 +107,22 @@ export default function ArticleReader({ api, teacherArticleID }) {
   };
 
   const handleScroll = () => {
-    onScroll(api, articleID, UMR_SOURCE);
+    let percentage = Math.floor(updateScrollPosition() * 100);
+    let currentSessionDuration = activeSessionDurationRef.current;
+    if (
+      currentSessionDuration - lastSampleScroll.current >=
```

Comment:
this is hard to read. 

could it be that the names that would make it more readable if they would be both talking about "timers"? e.g. 

```js
currentSessionTimer - lastSampleScrollTimer.current 
```

I know I started with the `duration` but even that one might benefit from being renamed... the suggestion is in the name of the hook too... it would then become: 
 
```js
const [activeSessionTimer, clockActive] = useActivityTimer(uploadActivity);
```

even better, probably in the ArticleReader it could become `readingSessionTimer` ?

Reply:
I decided to just change the named inside the function, but it could be good to do it across. I think it can be nice that the same component is named the same across all the code though. If it is `activeSessionDuration` becomes `readingSessionTimer` it might be confusing if I don't check that it is the same component.